go.mod

```diff
@@ -1,11 +1,10 @@
-module github.com/cisco-app-networking/nsm-nse
+module cisco-app-networking.github.io/nsm-nse
 
 go 1.14
 
 require (
-	github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd // indirect
+	github.com/cisco-app-networking/nsm-nse v0.2.5-vl3
```

Comment:
I think latest in v0.2.6-vl3, but anyway I don't understand what is the problem that we are trying to solve by adding this?
It is a sort of saying that a package depends on itself (well a previous/past version of itself). Could you elaborate more pls.